sockeye/__init__.py

```diff
@@ -11,4 +11,4 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-__version__ = '1.8.4'
+__version__ = '1.8.5'
```

Comment:
Let's bump the major version and go to '1.9.0' as the Translator interface changes.